priv/sites/zotonic_status/modules/mod_zotonic_status_vcs/mod_zotonic_status_vcs.erl

```diff
@@ -75,10 +75,12 @@ event(#postback{message=make}, Context) ->
 
 
 % @doc Show a notice on the current webpage.
+-spec show_notice(atom(), string(), #context{}) -> #context{}.
 show_notice(SiteName, Text, Context) ->
     z_session_page:add_script(notice(SiteName, Text, Context)).
 
 % @doc Render a notice.
+-spec notice(binary(), string(), #context{}) -> #context{}.
```

Comment:
The spec in line 83 uses a record, please define a type for the record and use that instead.